.ci/redmine_matrix_job.yaml

```diff
@@ -0,0 +1,59 @@
+---
+job: LibVMA-redmine-issue
+
+registry_host: harbor.mellanox.com
+registry_auth: swx-storage
+
+kubernetes:
+  privileged: true
+  cloud: swx-k8s-spray
```

Comment:
lets use blossom k8s 
  cloud: il-ipp-blossom-prod
  namespace: swx-media

Reply:
Swapped to Blossom k8s